src/syslog_mcp/services/alert_monitor.py

```
@@ -0,0 +1,357 @@
+"""
+Alert monitoring service that evaluates alert rules and sends notifications.
+"""
+
+import asyncio
+import json
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)